src/argdialog.cpp

```diff
@@ -295,6 +295,9 @@ void ArgDialog::setFilePath(QString path) { m_file_path = path; }
 void ArgDialog::setConfig(QString config_string) {
   m_config_string = config_string;
   ui->user->addItems(getValueArgs(m_config_string.toUtf8(), "users"));
+  if (ui->user->count() > 0) {
+    m_ig_post.user = ui->user->itemText(0).toUtf8().constData();
```

Comment:
Todo: only allow adding users once Send button is hit.

Reply:
https://github.com/StronglogicSolutions/ky_gui/issues/22